docs/sources/setup/migrate/ssd-to-distributed/_index.md

```diff
@@ -0,0 +1,372 @@
+
+## Prerequisites
+
+Before starting the migration process, make sure you have the following prerequisites:
+
+1. Access to your Kubernetes cluster via `kubectl`.
+1. Helm Installed
+
+## SSD Example
+
+This example will use the following SSD deployment as a reference:
+
+```yaml
+---
+loki:
+   schemaConfig:
+     configs:
+       - from: "2024-04-01"
+         store: tsdb
+         object_store: s3
+         schema: v13
+         index:
+           prefix: loki_index_
+           period: 24h
+   storage_config:
+     aws:
+       region: eu-central-1
+       bucketnames: aws-chunks-bucket
+       s3forcepathstyle: false
+   ingester:
+       chunk_encoding: snappy
+   ruler:
+    enable_api: true
+    storage:
+      type: s3
+      s3:
+        region: eu-central-1
+        bucketnames: aws-ruler-bucket
+        s3forcepathstyle: false
+      alertmanager_url: http://prom:9093
+   querier:
+      max_concurrent: 4
+
+   storage:
+      type: s3
+      bucketNames:
+        chunks: "aws-chunks-bucket"
+        ruler: "aws-ruler-bucket"
+      s3:
+        region: eu-central-1
+
+deploymentMode: SimpleScalable
+
+# SSD
+backend:
+ replicas: 2
+read:
+ replicas: 3
+write:
+ replicas: 3
+
+# Distributed Loki
+ingester:
+ replicas: 0
+ zoneAwareReplication:
+  enabled: false
+
+querier:
+ replicas: 0
+ maxUnavailable: 0
+queryFrontend:
+ replicas: 0
+ maxUnavailable: 0
+queryScheduler:
+ replicas: 0
+distributor:
+ replicas: 0
+ maxUnavailable: 0
+compactor:
+ replicas: 0
+indexGateway:
+ replicas: 0
+ maxUnavailable: 0
+ruler:
+ replicas: 0
+ maxUnavailable: 0
+
+# Single binary Loki
+singleBinary:
+ replicas: 0
+
+minio:
+ enabled: false
+```
+
+{{< admonition type="note" >}}
+Note: that this example is only a reference on the parameters that need to be changed. There will be other parameters within your own config such as `limits_config`, `gateway`, `compactor`, etc. These can remain the same.
+{{< /admonition >}}
+
+## Stage 1: Deploying the Loki Distributed Components
+
+In this stage, we will deploy the distributed Loki components alongside the SSD components. We will also change the `deploymentMode` to `SimpleScalable<->Distributed`. The `SimpleScalable<->Distributed` migration mode allows for a zero-downtime transition between SimpleScalable and fully Distributed architectures. During migration, both deployment types run simultaneously, sharing the same object storage backend.
+
+```
+SimpleScalable Components   ->    Distributed Components
+------------------------    ->    ---------------------
+write (StatefulSet)         ->    distributor + ingester
+read (Deployment)           ->    query-frontend + querier
+backend (StatefulSet)       ->    compactor + ruler + index-gateway
```

Comment:
Let's match how we spell/capitalize these in the Components and Deployment modes topics.
```suggestion
write (StatefulSet)         ->    Distributor + Ingester
read (Deployment)           ->    Query Frontend + Querier
backend (StatefulSet)       ->    Compactor + Ruler + Index Gateway
```